infoschema/perfschema/const.go

```diff
@@ -549,3 +550,10 @@ const tablePDProfileGoroutines = "CREATE TABLE IF NOT EXISTS " + tableNamePDProf
 const tableSessionVariables = "CREATE TABLE IF NOT EXISTS " + tableNameSessionVariables + " (" +
 	"VARIABLE_NAME VARCHAR(64) NOT NULL," +
 	"VARIABLE_VALUE VARCHAR(1024) NOT NULL);"
+
+// tableSessionConnectAttrs contains the column name definitions for the table session_connect_attrs
+const tableSessionConnectAttrs = "CREATE TABLE IF NOT EXISTS " + tableNameSessionConnectAttrs + " (" +
+	"PROCESSLIST_ID bigint unsigned NOT NULL," +
+	"ATTR_NAME varchar(32) COLLATE utf8mb4_bin NOT NULL," +
+	"ATTR_VALUE varchar(1024) COLLATE utf8mb4_bin DEFAULT NULL," +
+	"ORDINAL_POSITION int DEFAULT NULL);"
```

Comment:
I notice MySQL [session_connect_attrs](https://dev.mysql.com/doc/refman/8.0/en/performance-schema-session-connect-attrs-table.html) has definied PK. why we here do not need PK for the table?

Reply:
It looks like none of the I_S and P_S tables have a primary key, so I followed that. Not sure if that's required or not. I also don't think having a PK makes a big difference here?

```
sql> select * from information_schema.KEY_COLUMN_USAGE WHERE CONSTRAINT_NAME='PRIMARY' AND table_schema='performance_schema';
Empty set (0.0048 sec)

sql> select * from information_schema.KEY_COLUMN_USAGE WHERE CONSTRAINT_NAME='PRIMARY' AND table_schema='information_schema';
Empty set (0.0042 sec)
```

Reply:
if there is a little rows in this table, it is no big difference.